src/ansys/mapdl/core/mapdl_grpc.py

```diff
@@ -610,9 +610,17 @@ def _channel_str(self):
         Generally of the form of "ip:port", like "127.0.0.1:50052".
 
         """
-        if self._channel is not None:
-            return self._channel._channel.target().decode()
-        return ""
+        channel = self._channel
+        while channel is not None:
+            # When creating interceptors, channels have a nested "_channel" member
+            # containing the intercepted channel.
+            # Only the actual channel contains the "target" member describing the address
+            if hasattr(channel, "target"):
+                return channel.target().decode()
+            channel = getattr(channel, "_channel", None)
+        # This method is relying on grpc channel's private attributes, fallback in case
+        # it does not exist
+        return "unknown"  #  pragma: no cover Unreachable in the current gRPC version
```

Comment:
Returning a string? ... I guess it is a gRPC thing??

Reply:
I did not get this one, this property is called `channel_str`, so yep it returns a string :D

Reply:
fml.... Sorry.. it was my first day after holidays... :( 

https://www.youtube.com/watch?v=9ZlOhSt_qW0

Reply:
Lol no problem, good luck on readjusting